doc/changes/latest.inc

```diff
@@ -15,6 +15,8 @@ Current (0.21.dev0)
 Changelog
 ~~~~~~~~~
 
+- Modified :meth:`mne.channels.channels.UpdateChannelsMixin.pick_types` to remove dropped channel types from `reject` and `flat` dictionaries
```

Comment:
```suggestion
- Modified :meth:`:meth:`mne.Epochs.pick_types` to remove dropped channel types from `reject` and `flat` dictionaries by `Rahul Nadkarni`_
```

Reply:
I can make this change, but removing keys from `reject` and `flat` in `UpdateChannelsMixin.pick_types` should apply to other objects (e.g., `Raw`, `Evoked`) as well right?

I'll definitely add my name as well, thanks for catching that!